.github/workflows/php.yml

```diff
@@ -0,0 +1,90 @@
+name: PHP Linting and Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - 'feature/**'
+      - 'release/**'
+
+jobs:
+  lint:
+    name:    PHP Linting
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+      - uses: actions/cache@v2
+        with:
+          path: vendor/
+          key: ${{ runner.os }}-vendor-${{ hashFiles('composer.lock') }}
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.3'
+          tools:       composer
+          coverage:    none
+      - name: Set up branches
+        run: git checkout -b master refs/remotes/origin/master && git checkout -
+      - name: Install PHP dependencies
+        run: composer self-update --1 && composer install --no-ansi --no-interaction --prefer-dist --no-progress
+      - name: Check for new issues
+        run: ./scripts/linter-ci
+      - name: Check for escaping
+        run: ./vendor/bin/phpcs -s --sniffs=WordPress.WP.I18n,Generic.PHP.Syntax,WordPress.Security.EscapeOutput .
+      - name: Check for nonces
+        run: ./vendor/bin/phpcs -sn --sniffs=WordPress.Security.NonceVerification .
+      - name: Check WPCOM rules
+        run: ./vendor/bin/phpcs -sn --standard=./wpcom-phpcs.xml .
+  test:
+    name: PHP Unit Tests
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+      max-parallel: 10
+      matrix:
+        wp: [ 'latest' ]
+        wpmu: [ 0 ]
```

Comment:
I think that we had a run with multisite previously. Was this removed for a reason?

Reply:
I do it manually below with just one of the PHP versions (`include:`).

Reply:
Ah cool! I was right to suspect that I was missing something 🙂